2014-05-09 18:17:29,424 - [INFO] - from play in pool-21-thread-4 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2014-05-09 18:17:34,220 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:17:34,887 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-05-09 18:17:39,903 - [ERROR] - from play in play-internal-execution-context-2 
Cannot invoke the action, eventually got an error: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException

2014-05-09 18:17:39,917 - [ERROR] - from application in play-internal-execution-context-2 


! @6i6m56kja - Internal server error, for (POST) [/login] ->

play.api.Application$$anon$1: Execution exception[[RuntimeException: java.lang.reflect.InvocationTargetException]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) [play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at play.data.Form.bind(Form.java:391) ~[play-java_2.10-2.2.2.jar:2.2.2]
	at play.data.Form.bindFromRequest(Form.java:215) ~[play-java_2.10-2.2.2.jar:2.2.2]
	at controllers.Login.authenticate(Login.java:108) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$15$$anonfun$apply$15.apply(routes_routing.scala:333) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$15$$anonfun$apply$15.apply(routes_routing.scala:333) ~[classes/:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.GlobalSettings$1.call(GlobalSettings.java:64) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10-2.2.2.jar:2.2.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_45]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:1.7.0_45]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:1.7.0_45]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[na:1.7.0_45]
	at play.data.Form.bind(Form.java:388) ~[play-java_2.10-2.2.2.jar:2.2.2]
	... 22 common frames omitted
Caused by: javax.persistence.PersistenceException: Error loading on models.User.active
	at com.avaje.ebeaninternal.server.query.SqlBeanLoad.load(SqlBeanLoad.java:113) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.deploy.BeanProperty.load(BeanProperty.java:587) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.SqlTreeNodeBean.load(SqlTreeNodeBean.java:257) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readRow(CQuery.java:497) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readBeanInternal(CQuery.java:531) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.hasNextBean(CQuery.java:658) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readTheRows(CQuery.java:645) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readCollection(CQuery.java:611) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQueryEngine.findMany(CQueryEngine.java:172) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.DefaultOrmQueryEngine.findMany(DefaultOrmQueryEngine.java:77) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.OrmQueryRequest.findList(OrmQueryRequest.java:265) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.findList(DefaultServer.java:1460) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.findUnique(DefaultServer.java:1230) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.querydefn.DefaultOrmQuery.findUnique(DefaultOrmQuery.java:903) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.util.DefaultExpressionList.findUnique(DefaultExpressionList.java:193) ~[avaje-ebeanorm.jar:na]
	at models.User.findByLogin(User.java:43) ~[classes/:na]
	at controllers.Login.validarUser(Login.java:41) ~[classes/:na]
	at controllers.Login.validate(Login.java:21) ~[classes/:na]
	... 27 common frames omitted
Caused by: java.lang.RuntimeException: setIntercept active on [models.User] arg[null] type[models.User] threw error
	at com.avaje.ebeaninternal.server.deploy.BeanProperty.setValueIntercept(BeanProperty.java:726) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.SqlBeanLoad.load(SqlBeanLoad.java:103) ~[avaje-ebeanorm.jar:na]
	... 44 common frames omitted
Caused by: java.lang.NullPointerException: null
	at models.User._ebean_setFieldIntercept(User.java:1) ~[classes/:na]
	at com.avaje.ebeaninternal.server.reflect.EnhanceBeanReflect$Setter.setIntercept(EnhanceBeanReflect.java:184) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.deploy.BeanProperty.setValueIntercept(BeanProperty.java:716) ~[avaje-ebeanorm.jar:na]
	... 45 common frames omitted

2014-05-09 18:17:46,761 - [ERROR] - from play in play-internal-execution-context-2 
Cannot invoke the action, eventually got an error: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException

2014-05-09 18:17:46,765 - [ERROR] - from application in play-internal-execution-context-2 


! @6i6m574mo - Internal server error, for (POST) [/login] ->

play.api.Application$$anon$1: Execution exception[[RuntimeException: java.lang.reflect.InvocationTargetException]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) [play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at play.data.Form.bind(Form.java:391) ~[play-java_2.10-2.2.2.jar:2.2.2]
	at play.data.Form.bindFromRequest(Form.java:215) ~[play-java_2.10-2.2.2.jar:2.2.2]
	at controllers.Login.authenticate(Login.java:108) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$15$$anonfun$apply$15.apply(routes_routing.scala:333) ~[classes/:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$15$$anonfun$apply$15.apply(routes_routing.scala:333) ~[classes/:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.GlobalSettings$1.call(GlobalSettings.java:64) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10-2.2.2.jar:2.2.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted
Caused by: java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_45]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:1.7.0_45]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:1.7.0_45]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[na:1.7.0_45]
	at play.data.Form.bind(Form.java:388) ~[play-java_2.10-2.2.2.jar:2.2.2]
	... 22 common frames omitted
Caused by: javax.persistence.PersistenceException: Error loading on models.User.active
	at com.avaje.ebeaninternal.server.query.SqlBeanLoad.load(SqlBeanLoad.java:113) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.deploy.BeanProperty.load(BeanProperty.java:587) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.SqlTreeNodeBean.load(SqlTreeNodeBean.java:257) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readRow(CQuery.java:497) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readBeanInternal(CQuery.java:531) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.hasNextBean(CQuery.java:658) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readTheRows(CQuery.java:645) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQuery.readCollection(CQuery.java:611) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.CQueryEngine.findMany(CQueryEngine.java:172) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.DefaultOrmQueryEngine.findMany(DefaultOrmQueryEngine.java:77) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.OrmQueryRequest.findList(OrmQueryRequest.java:265) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.findList(DefaultServer.java:1460) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.findUnique(DefaultServer.java:1230) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.querydefn.DefaultOrmQuery.findUnique(DefaultOrmQuery.java:903) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.util.DefaultExpressionList.findUnique(DefaultExpressionList.java:193) ~[avaje-ebeanorm.jar:na]
	at models.User.findByLogin(User.java:43) ~[classes/:na]
	at controllers.Login.validarUser(Login.java:41) ~[classes/:na]
	at controllers.Login.validate(Login.java:21) ~[classes/:na]
	... 27 common frames omitted
Caused by: java.lang.RuntimeException: setIntercept active on [models.User] arg[null] type[models.User] threw error
	at com.avaje.ebeaninternal.server.deploy.BeanProperty.setValueIntercept(BeanProperty.java:726) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.query.SqlBeanLoad.load(SqlBeanLoad.java:103) ~[avaje-ebeanorm.jar:na]
	... 44 common frames omitted
Caused by: java.lang.NullPointerException: null
	at models.User._ebean_setFieldIntercept(User.java:1) ~[classes/:na]
	at com.avaje.ebeaninternal.server.reflect.EnhanceBeanReflect$Setter.setIntercept(EnhanceBeanReflect.java:184) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.deploy.BeanProperty.setValueIntercept(BeanProperty.java:716) ~[avaje-ebeanorm.jar:na]
	... 45 common frames omitted

2014-05-09 18:23:25,034 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:23:25,188 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-05-09 18:23:55,059 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:23:55,209 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-05-09 18:24:42,830 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:24:42,979 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-05-09 18:45:27,304 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:45:27,457 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-05-09 18:46:02,785 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:46:02,937 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-05-09 18:46:24,569 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:46:24,719 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-05-09 18:46:44,567 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:46:44,717 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-05-09 18:47:48,758 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:47:48,910 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-05-09 18:48:12,362 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:48:12,512 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-05-09 18:49:40,036 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:49:40,190 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-05-09 18:53:01,942 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:53:02,096 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2014-05-09 18:54:42,233 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:54:42,379 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 18:59:41,779 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 18:59:41,934 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:04:00,655 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:04:00,803 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:08:43,404 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:08:43,554 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:09:12,265 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:09:12,415 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:09:41,578 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:09:41,726 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:11:00,700 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:11:00,850 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:11:35,160 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:11:35,310 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:14:25,217 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:14:25,368 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:17:14,803 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:17:14,951 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:18:05,935 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:18:06,081 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2014-05-09 19:18:43,293 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:18:43,441 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2014-05-09 19:20:54,931 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:20:55,076 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2014-05-09 19:24:07,299 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:24:07,465 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2014-05-09 19:25:26,201 - [INFO] - from play in play-internal-execution-context-20 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:25:26,365 - [INFO] - from play in play-internal-execution-context-20 
Application started (Dev)

2014-05-09 19:26:25,208 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:26:25,362 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2014-05-09 19:27:50,357 - [ERROR] - from application in New I/O worker #5 


! @6i6mec95e - Internal server error, for (GET) [/bars/hdi] ->

play.PlayExceptions$CompilationException: Compilation error[error: unreachable statement]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-05-09 19:28:04,486 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:28:04,634 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2014-05-09 19:29:03,835 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:29:03,984 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2014-05-09 19:29:30,044 - [INFO] - from play in play-internal-execution-context-24 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:29:30,194 - [INFO] - from play in play-internal-execution-context-24 
Application started (Dev)

2014-05-09 19:31:20,909 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:31:21,073 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2014-05-09 19:32:38,712 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:32:38,863 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2014-05-09 19:42:39,783 - [INFO] - from play in play-internal-execution-context-27 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:42:39,938 - [INFO] - from play in play-internal-execution-context-27 
Application started (Dev)

2014-05-09 19:47:33,252 - [INFO] - from play in play-internal-execution-context-29 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:47:33,402 - [INFO] - from play in play-internal-execution-context-29 
Application started (Dev)

2014-05-09 19:48:27,315 - [INFO] - from play in play-internal-execution-context-31 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:48:27,465 - [INFO] - from play in play-internal-execution-context-31 
Application started (Dev)

2014-05-09 19:49:15,541 - [INFO] - from play in play-internal-execution-context-31 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:49:15,693 - [INFO] - from play in play-internal-execution-context-31 
Application started (Dev)

2014-05-09 19:49:52,079 - [INFO] - from play in play-internal-execution-context-32 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:49:52,229 - [INFO] - from play in play-internal-execution-context-32 
Application started (Dev)

2014-05-09 19:50:55,560 - [INFO] - from play in play-internal-execution-context-32 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:50:55,711 - [INFO] - from play in play-internal-execution-context-32 
Application started (Dev)

2014-05-09 19:52:41,910 - [INFO] - from play in play-internal-execution-context-32 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:52:42,063 - [INFO] - from play in play-internal-execution-context-32 
Application started (Dev)

2014-05-09 19:54:04,743 - [INFO] - from play in play-internal-execution-context-32 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:54:04,896 - [INFO] - from play in play-internal-execution-context-32 
Application started (Dev)

2014-05-09 19:54:52,995 - [INFO] - from play in play-internal-execution-context-34 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:54:53,152 - [INFO] - from play in play-internal-execution-context-34 
Application started (Dev)

2014-05-09 19:56:00,628 - [INFO] - from play in play-internal-execution-context-34 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:56:00,781 - [INFO] - from play in play-internal-execution-context-34 
Application started (Dev)

2014-05-09 19:57:22,575 - [INFO] - from play in play-internal-execution-context-34 
database [default] connected at jdbc:h2:mem:play

2014-05-09 19:57:22,778 - [INFO] - from play in play-internal-execution-context-34 
Application started (Dev)

